autoload/lumen/platforms/macos/watcher.swift

```diff
@@ -5,6 +5,8 @@
 import Cocoa
 import Darwin
 
+setbuf(stdout, nil); // don't buffer the output. thats why print() didn't work.
```

Comment:
Awesome, thanks for finding the real issue with `print()`. I think we could also have fixed the issue by attaching a real pseudo tty in the Job api, but I prefer this solution.